introduction_to_amazon_algorithms/jumpstart_image_classification/README.md

```diff
@@ -1,2 +1,2 @@
 ### SageMaker JumpStart Image classification Training & Deployment
-This notebook `Amazon_JumpStart_Image_Classification.ipynb` demos how to fine-tune and deploy a pre-trained image classification model using JumpStart API. It shows how to select a pre-trained image classification model from JumpStart and fine-tune it on an example dataset containing raw .jpg/.png images, while varying training hyperparameters such as learning rate, batch-size and number of epochs. Once the training is complete, the notebook shows how to host the trained model for inference. It also shows how to host the pre-trained model as-it-is without first fine-tuning it.
+This notebook `Amazon_JumpStart_Image_Classification.ipynb` demos how to fine-tune and deploy a pre-trained image classification model using JumpStart API. It shows how to select a pre-trained image classification model from JumpStart and fine-tune it on an example dataset containing raw .jpg/.png images, while varying training hyperparameters such as learning rate, batch-size and number of epochs. AMT is used to search for the best hyperparameters. Once the training is complete, the notebook shows how to host the trained model for inference. It also shows how to host the pre-trained model as-it-is without first fine-tuning it.
```

Comment:
State the full term before/instead of abbreviating "AMT" in the 4 readme files, ie: `Automatic Machine Translation (AMT) is used...` or `Automatic Machine Translation is used...`

Reply:
Since AMT is the most known name, I will change this to AMT (Automatic Model Tuning).